source/slang/slang-serialize-ir-types.h

```diff
@@ -129,7 +129,7 @@ struct IRSerialData
         
         SLANG_FORCE_INLINE bool operator!=(const ThisType& rhs) const { return !(*this == rhs); }
 
-        uint8_t m_op;                       ///< For now one of IROp 
+        uint16_t m_op;                      ///< For now one of IROp 
```

Comment:
In order to keep the current packing, it would be better to make m_pad0 into a uint8_t.